src/views/routes/sauna/SaunaEditorView.tsx

```diff
@@ -0,0 +1,37 @@
+import { useParams } from 'react-router-dom'
```

Comment:
Editing seems not to work for me for latest backend (manually disabled authentication for the route to test it):
```
Sauna.ts:68 Uncaught (in promise) Error: Object [object Object] could not be mapped in. It is not of type RemoteResponse.
    at Object.mapIn (Sauna.ts:68:1)
    at Object.get (sauna.ts:22:1)
```

Reply:
I tested it manually, should work now